screens/portfolio/index.tsx

```diff
@@ -0,0 +1,179 @@
+import { StyleSheet, Text, View } from 'react-native';
+import React, { useState } from 'react';
+import { LQDActionCard, LQDBottomSheet, LQDButton, LQDFlatlist, LQDScrollView, LQDSlider, LQDStrategyCard } from '@/components';
+import { adjustFontSizeForIOS } from '@/utils/helpers';
+import { addMoney, strategyies } from '../discover/dummy';
+import { IActionIconVariant } from '@/components/action-card/types';
+import useSystemFunctions from '@/hooks/useSystemFunctions';
+import AssetItem from './asset-item';
+
+const Portfolio = () => {
+  const [showModal, setShowModal] = useState(false);
+  const [selected, setSelected] = useState('');
```

Comment:
Please work on your variable names. `selected` is ambiguous. What is being selected?